src/Bicep.Core.UnitTests/Diagnostics/LinterAnalyzerTests.cs

```diff
@@ -34,17 +34,8 @@ public void HasBuiltInRules()
         [DataRow(ExplicitValuesForLocationParamsRule.Code)]
```

Comment:
No need to have a complete list here that people need to update each time.  Restricting to small list for sanity check.